src/utilities/use-component-did-mount.ts

```diff
@@ -3,10 +3,20 @@ import {EffectCallback} from '../types';
 import {useIsAfterInitialMount} from './use-is-after-initial-mount';
 
 /**
- * Similarly to the lifecycle hook componentDidMount, useComponentDidMount
+ * Similarly to the life-cycle method componentDidMount, useComponentDidMount
  * will be invoked after the component has mounted, and only the initial mount.
  * @param callback Defines a callback to invoke once the component has
  * initially mounted.
+ * @example
+ * function Playground({active}) {
+ *  useComponentDidMount(() => {
```

Comment:
Would it be worth expanding on why we would use this vs `useEffect` with an empty dependency array?

Reply:
These docs were more meant to explain the hook usage, rather than the trade-offs between react API's. I'm going to leave it out for now, but if this comes up again we can look into adding a reference to the use-effect docs 😄